features/post.feature

```diff
@@ -66,7 +66,7 @@ Feature: Manage WordPress posts
       | post_name  |           |
       | post_type  | post      |
 
-    When I run `wp post get {POST_ID} --format=csv --fields=post_title,type | wc -l`
+    When I run `wp post get {POST_ID} --format=csv --fields=post_title,type | wc -l | tr -d ' '`
```

Comment:
For general information (don't think it actually needs to be changed), you can also pass `--format=count` to `wp post get`. Ditto any other command that supports `--format=<format>`